content/docs/preferences.md

```diff
@@ -517,6 +517,12 @@ The following settings are for rather specific needs and could cause some confus
 ```
 Simply add `"regexBackward4PowerUser"="yes"` if this option is absent.
 
+* Change the number of expressions that can be saved in the [Find/Replace](../searching/) dialog's Find, Replace, Filters, and Directory fields.  They are set in the `<FindHistory ...>` tag inside `config.xml`, using the attributes:
+    * `nbMaxFindHistoryFind` => changes the number of **Find what** entries that are saved in the history (0 .. 30, default = 10)
+    * `nbMaxFindHistoryReplace` => changes the number of **Replace with** entries that are saved in the history (0 .. 30, default = 10)
+    * `nbMaxFindHistoryPath` => changes the number of **Filters** entries that are saved in the history (0 .. 30, default = 10)
+    * `nbMaxFindHistoryFilter` => changes the number of **Directory** entries that are saved in the history (0 .. 30, default = 10)
```

Comment:
Here's where they are defined: https://github.com/notepad-plus-plus/notepad-plus-plus/blob/f9d6fb9e31719a0ef9bbc8bd42c262560575adf8/PowerEditor/src/Parameters.h#L982

Note that the default for the filters is 20, not 30 as you have here in the docs.

Reply:
per https://github.com/notepad-plus-plus/notepad-plus-plus/blob/af56713079c9e31b8c81b5603a6c841afb3e2005/PowerEditor/src/config.4zipPackage.xml#L50 , the default value in the config file is 10 for all four.  The line you point to actually indicates that for the filter, the MAX value is 20.  Unless I've read things wrong.

Reply:
My brain is gone:  I meant to say "max" not "default".
The max for the filters is the only one of the four that is not 30; it is 20; that's what my source code reference was about.
Your preferences.md change has 30 for all four.  That was the complaint.